index.js

```diff
@@ -173,7 +172,7 @@ module.exports = function(version, _options) {
                 .replace('{lane_instruction}', laneInstruction)
                 .replace('{modifier}', instructions[language][version].constants.modifier[modifier])
                 .replace('{direction}', this.directionFromDegree(language, step.maneuver.bearing_after))
-                .replace('{nth}', nthWaypoint)
+                .replace('{nth}', legIndex && legIndex > 0 ? this.ordinalize(language, legIndex) : '')
```

Comment:
According to this logic, if there are multiple destinations, the first destination is announced as, “You have arrived at your destination.”